python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -840,97 +840,133 @@ async def arun(self, checkpoint: bool = False) -> None:
         self.offset_total_focus = 0.0
         self.offset_total_coma_x = 0.0
         self.offset_total_coma_y = 0.0
-        for i in range(self.max_iter):
-            self.log.debug(f"CWFS iteration {i + 1} starting...")
 
-            if checkpoint:
-                await self.checkpoint(
-                    f"[{i + 1}/{self.max_iter}]: CWFS loop starting..."
-                )
-
-            # Setting visit_id's to none so run_cwfs will take a new dataset.
-            self.intra_visit_id = None
-            self.extra_visit_id = None
-            await self.take_intra_extra()
-
-            results = await self.run_align()
+        # Getting initial hexapod position
+        self.initial_hexapod_position = await self.latiss.get_hexapod_position()
+        try:
```

Comment:
I think that rather than doing a try/except here, you could handle this in the `cleanup` method. This is a coroutine that is automatically called when the script is terminating. 

Take a look at how the cleanup is used in the [base_track_target_and_take_image](https://github.com/lsst-ts/ts_standardscripts/blob/05ad34eee66b7e7b3f29bf6ea5b301cdd9ce0e36/python/lsst/ts/standardscripts/base_track_target_and_take_image.py#L290) script.

Reply:
Also, check the documentation on the cleanup method [here](https://ts-salobj.lsst.io/sal_scripts.html#cleanup-method-optional).